src/github.py

```diff
@@ -8,6 +8,30 @@ def __init__(self, group: str, repo: str, auth_token: str = None) -> None:
         self.repo: str = repo
         self.auth_token: str = auth_token
 
+    def get_test_result(test, logfiles, col):
+        test_exist = False
+        test_col = ':green_circle:'
+        for i in range(len(logfiles)):
+            if test in logfiles[i]:
+                test_exist = True
+                if col[i] == ':red_circle:' or col[i] == ':lock:' or col[
+                        i] == ':wastebasket:' or col[i] == ':hourglass:':
+                    test_col = ':red_circle:'
+                elif col[
+                        i] == ':yellow_circle:' and not test_col == ':red_circle:':
+                    test_col = ':yellow_circle:'
+        return (test_exist, test_col)
+
+    def add_test_to_text(test, test_exist, test_col, col, logfiles, text):
+        if test_exist:
+            details = [['', 'Test']]
+            for i in range(len(logfiles)):
```

Comment:
`for c, logfile in zip(col, logfiles):` is more pythonic

Reply:
Thanks, that looks better!